urllib3/util/connection.py

```diff
@@ -34,7 +37,8 @@ def is_connection_dropped(conn):  # Platform-specific
 # One additional modification is that we avoid binding to IPv6 servers
 # discovered in DNS if the system doesn't have IPv6 functionality.
 def create_connection(address, timeout=socket._GLOBAL_DEFAULT_TIMEOUT,
-                      source_address=None, socket_options=None):
+                      source_address=None, socket_options=None,
+                      enable_happy_eyeballs=True):
```

Comment:
Sorry if missing something here, but is there a reason for both the global `ENABLE_HAPPY_EYEBALLS` and the `create_connection` param `enable_happy_eyeballs`? It currently looks like only the global is being used.